custom-vu/src/main/kotlin/jces1209/vu/page/issue/CloudProjectIssueNavigatorPage.kt

```diff
@@ -0,0 +1,46 @@
+package jces1209.vu.page.issue
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import jces1209.vu.page.issue.AbstractProjectIssueNavigatorPage
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+import java.time.Duration
+
+class CloudProjectIssueNavigatorPage(
+    private val driver: WebDriver
+) : AbstractProjectIssueNavigatorPage {
+    override fun openProjectByIndex(numberOfProject: String) {
```

Comment:
`number` and `String` have incompatible meanings.
Let's use the Kotlin/Java typesystem to our advantage.
```suggestion
    override fun openProjectByIndex(index: Int) {
```